app/content/blog/building-beyond-beginner-guitar.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "Building 'Beyond Beginner Guitar' on Urbit"
+date = "2025-12-07"
+description = "~nordus-mocwyl walks through what it took to build an guitar course and memeber community on Urbit"
+# aliases = []
+
+[extra]
+# author = ""
+ship = "~nordus-mocwyl"
+image = "https://s3.us-east-1.amazonaws.com/urbit.orgcontent/Blog/Blog_QA+mastyr-bottec_Social.png"
+imageCard = "https://s3.us-east-1.amazonaws.com/urbit.orgcontent/Blog/Blog_QA+mastyr-bottec_Social.png"
+imageIndex = "https://s3.us-east-1.amazonaws.com/urbit.orgcontent/Blog/Blog_QA+mastyr-bottec_Banner.png"
+# imageDark =
+# imageCardDark =
+# imageIndexDark =
+tags =  ["courseware", "hawk", "userspace"]
++++
+
+
+By way of introduction, I am `~nordus-mocwyl`. While my background is in music, with credentials including a degree in guitar performance, I also am a graduate of [Hoon School](https://docs.urbit.org/build-on-urbit/hoon-school) Live and [App School](https://docs.urbit.org/build-on-urbit/app-school) Live, and former Urbit Foundation grantee. Urbit captured my attention as a way to own the full networked computing stack, and be able to create digital experiences that I control from top to bottom. One such experience is the recent ['Beyond Beginner Guitar'](https://howmbase.com/demo) course that I created. It is my own little 'Urbit powered business'. And while it will continue to evolve (and become more fully urbit-native), it works *today* and it doesn't demand my create life energy to be stripe mined, cataloged, quantified, and sold to the highest bidder.
```

Comment:
Last sentence:
And while it will continue to evolve (and become more fully urbit-native), it works *today* and it doesn't demand my create life energy to be stripe mined, cataloged, quantified, and sold to the highest bidder.

Change to:
While it will continue to evolve (and become more fully urbit-native), it works *today* and it doesn't demand my creative life energy to be strip-mined, cataloged, quantified, and sold to the highest bidder.

- creative
- strip-mined